### PR TITLE
feat: Add PR_BODY option

### PR DESCRIPTION
**Pull request by dhruvkb**
This PR adds a PR body input to the action, that can be used to add additional text to the PR description. This is useful when trying the explain more about the PR such as the cause behind its creation and how the future changes will lead to a forced push instead of a new PR.

Originally discussed in #78.

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -44,6 +44,10 @@ try {
 			type: 'array',
 			disableable: true
 		}),
+		PR_BODY: getInput({
+			key: 'PR_BODY',
+			default: [ '' ]
```
Comment (BetaHuhn) [inline src/config.js:49]: Why make the default an array? Just use the empty string directly: `default: ''`

File: src/git.js [hunk 2]
```diff
@@ -196,6 +197,8 @@ class Git {
 
 			${ changedFiles }
 
+			${ PR_BODY }
```
Comment (BetaHuhn) [inline src/git.js:200]: I think it might be better to insert it above the changed files. That way it's easier to see in the actual PR.

**Review by BetaHuhn — CHANGES_REQUESTED** [review]
Thanks for the great PR!

If you address the requested changes that would be great. Will merge afterwards!

File: src/git.js (hunk 2 above)
Comment (dhruvkb) [reply src/git.js:198]: Resolved in ba26cac.

File: src/config.js (hunk 1 above)
Comment (dhruvkb) [reply src/config.js]: Resolved in c5d2db7.

**Review by BetaHuhn — APPROVED** [review]
Great, thanks!

**Pull request merged**